src/shapes/Polyline.ts

```diff
@@ -103,11 +100,36 @@ export class Polyline<
    */
   constructor(points: XY[] = [], options: Props = {} as Props) {
     super({ points, ...options });
-    const { left, top } = options;
+    const { left, top, fromSVG } = options;
     this.initialized = true;
-    this.setBoundingBox(true);
-    typeof left === 'number' && this.set('left', left);
-    typeof top === 'number' && this.set('top', top);
+    const {
+      left: calculatedLeft,
+      top: calculatedTop,
+      width,
+      height,
+      pathOffset,
+      strokeOffset,
+    } = this._calcDimensions();
+    this.set({ width, height, pathOffset, strokeOffset });
+    if (fromSVG) {
+      // if coming from SVG just assign top and left, those will be fixed
+      // later by remove removeTransformMatrixForSvgParsing using the weird
+      // _findCenterFromElement.
+      // This is not clear and is probably wrong, but as of now it works in conjuction
+      // with that other parsing function
+      this.set({
+        left: calculatedLeft,
+        top: calculatedTop,
+      });
```

Comment:
same
should be handled by `fromElement`
Especially since we want some day to extract all SVG logic to a standalone